test/Calculations/test_StandardDeviation.py

```diff
@@ -0,0 +1,33 @@
+from src.Calculations.StandardDeviation import StandardDeviation, get_calculation
+from pytest import approx
+
+
+def test_get_calculation():
+    """
+    Make sure the get_calculation function returns the calculation
+    """
+    assert get_calculation() == StandardDeviation
+
+
+def test_standard_deviation():
+    """
+    Test functionality of the Average class
+    """
+    c = StandardDeviation()
+    test_data = [
+        [2, 4, 6, 8, 10],
+        [1, 3.14, 5, 7, 9],
+        [3, 3, 3, 3, 5, 100],
+        [8, 2, 1, 5],
+        [11, 13, 'twelve'],
+        []
+    ]
+    expected_result = [     # these are population std, not sample std. might be wrong depending on how numpy does std
```

Comment:
Can you make this note about it being population std in the `StandardDeviation.py` file? I think it is useful there as well